mvpa2/clfs/dummies.py

```diff
@@ -52,7 +52,7 @@ class RandomClassifier(Classifier):
     """Dummy classifier deciding on labels absolutely randomly
     """
 
-    __tags__ = ['random', 'non-deterministic']
+    __tags__ = ['random', 'non-deterministic', 'one-class-binary']
```

Comment:
it' not clear for me if the tag means that the classifier is supposed not to crash when given a one-class dataset, or if it makes any sense to be trained with such?